spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration.java

```diff
@@ -536,7 +535,7 @@ public void customize(FluentConfiguration configuration) {
 			Postgresql properties = this.properties.getPostgresql();
 			PropertyMapper map = PropertyMapper.get().alwaysApplyingWhenNonNull();
 			map.from(properties::getTransactionalLock)
-				.to(extension.via((ext, transactionalLock) -> ext.setTransactionalLock(transactionalLock)));
+				.to(extension.via(PostgreSQLConfigurationExtension::setTransactionalLock));
```

Comment:
Please revert these changes. We deliberately do not use method references when configuration Flyway to improve backwards- and forwards-compatibility (see https://github.com/spring-projects/spring-boot/issues/38164).